test/unit/gateways/pay_junction_v2_test.rb

```diff
@@ -205,6 +206,21 @@ def test_failed_store
     assert_match %r{Card Number is not a valid card number}, response.message
   end
 
+  def test_add_address
```

Comment:
This is a pretty cool way to test this!

Reply:
Credit goes to Leila! I just copied they way she implemented this in the Adyen gateway :)